src/utilities.cc

```diff
@@ -92,6 +95,11 @@ static void DebugWriteToStderr(const char* data, void *) {
   if (write(STDERR_FILENO, data, strlen(data)) < 0) {
     // Ignore errors.
   }
+#if defined(__ANDROID__)
+  __android_log_write(ANDROID_LOG_FATAL,
```

Comment:
As before: it is not clear why messages written to logcat are always of fatal severity.

Reply:
Because here is fatal error and is going to print stack trace.